packages/edit-navigation/src/store/selectors.js

```diff
@@ -25,6 +25,10 @@ import { KIND, POST_TYPE, buildNavigationPostId } from './utils';
  */
 export const getNavigationPostForMenu = createRegistrySelector(
 	( select ) => ( state, menuId ) => {
+		if ( menuId === null ) {
```

Comment:
Why is this needed? Could we instead avoid calling this selector at all when `menuId` is null? Or maybe add a brief comment above this condition :)

Reply:
True, 632437b does that 😄